modules/textures/src/lib/parsers/crunch-module-loader.js

```diff
@@ -11,6 +11,11 @@ export async function loadCrunchModule(options) {
   return loadCrunch(options);
 }
 
+/**
+ * Load crunch decoder module
+ * @param {any} options - Loader options
+ * @returns {Promise<any>} Promise of Module object
```

Comment:
Nit: Better type than any?

Reply:
It is too complicated. The module is pretty large object.